source/draw/gpu/opengl/OpenGLES3/ShaderProgram.ooc

```diff
@@ -157,7 +157,7 @@ ShaderProgram: class {
 	_glCompileShader: func (shaderID: UInt) { glCompileShader(shaderID) }
 	_compileShader: func (source: String, shaderID: UInt) -> Bool {
 		version(debugGL) { validateStart() }
-		glShaderSource(shaderID, 1, (source toCString())&, null)
+		glShaderSource(shaderID, 1, (String toConstCharPointerConstPointer((source toCString())&)), null)
```

Comment:
The obvious backdraw here is that a static function is used. It would be much better off as a member function, but I don't know how the macro should be written, then. Notably, the result of the macro must be used immediately, because if it passes through an ooc function on the way, it becomes a normal `char **` again. @sebastianbaginski, any ideas?